__mocks__/@react-native-firebase/crashlytics.ts

```diff
@@ -0,0 +1,20 @@
+import type {FirebaseCrashlyticsTypes} from '@react-native-firebase/crashlytics';
+
+// <App> uses <ErrorBoundary> and we need to mock the imported crashlytics module
+// due to an error that happens otherwise https://github.com/invertase/react-native-firebase/issues/2475
+
+type CrashlyticsModule = {
+    log: FirebaseCrashlyticsTypes.Module['log'];
+    recordError: FirebaseCrashlyticsTypes.Module['recordError'];
+    setCrashlyticsCollectionEnabled: FirebaseCrashlyticsTypes.Module['setCrashlyticsCollectionEnabled'];
+};
+
+type CrashlyticsMock = () => CrashlyticsModule;
+
+const crashlyticsMock: CrashlyticsMock = () => ({
```

Comment:
```suggestion
// <App> uses <ErrorBoundary> and we need to mock the imported crashlytics module
// due to an error that happens otherwise https://github.com/invertase/react-native-firebase/issues/2475
const crashlyticsMock: CrashlyticsMock = () => ({
```

Move the comment to here